src/components/People.js

```diff
@@ -0,0 +1,64 @@
+import { Component } from "react";
+import "../App.css";
+
+class People extends Component{
+
+    constructor(){
+        super();
+        this.state={
+            people:[],
+            searchInput:"",
+        }
```

Comment:
```suggestion
        this.state={
            people:[],
            searchInput:"",
            selectedPerson: null
        }
```
You need a third piece of state, `selectedPerson`, to keep track of which character should currently be displayed based on the user's search input.